app/models/slide.rb

```diff
@@ -92,7 +94,8 @@ def css_classes
     classes.join(' ')
   end
 
-  def take_screenshot
+  def take_screenshot(opt = false)
+    @skip_file_validation = opt
```

Comment:
This seems odd.  Calling take_screenshot, with that boolean probably shouldn't have the side effect of also setting a different attribute `@skip_file_validation`.  The `take_screentshot` should do only what it says its going to do.

If you need to skip_file_validation, set it explicitly somewhere else `@slide.skip_file_validation = true`.

Also, sandy metz says to not directly access the instance variable with the `@` sign.  Since you already defined attr_accessor, use the getters and settings automatically defined for you.  For example:
```ruby 
 # bad
def calculate_velocity
  @velocity = @speed * @distance
end


# good
attr_accessor :velocity, :speed, :distance

def calculate_velocity 
  self.velocity = speed * distance
end
```